app/packages/grits-net-meteor/client/templates/grits_confirm_modal.coffee

```diff
@@ -13,3 +13,6 @@ Template.gritsConfirmModal.events
   'click .load-all': (event, instance) ->
     GritsFilterCriteria.more(instance.data.callback, 0)
     instance.$('.confirm--modal').modal('hide')
+
+  # 'click .notify': (event, instance) ->
+  #   $('.notify-slider').slideToggle()
```

Comment:
We could - I was going to leave it because I think we are going to need it once we implement notifications for direct flights, but we don't have to.